broker/src/main/java/org/apache/rocketmq/broker/offset/LmqConsumerOffsetManager.java

```diff
@@ -36,6 +40,92 @@ public LmqConsumerOffsetManager(BrokerController brokerController) {
         super(brokerController);
     }
 
+    @Override
+    public boolean load() {
```

Comment:
IMO, compatibility in the ConfigManager would be better

Reply:
Compatibility logic can not remove to parent class, because it involves many different implementation methods, such as configFilePath, which can not obtain in parent class.